.github/workflows/failureNotifier.yml

```diff
@@ -0,0 +1,96 @@
+name: Notify on Workflow Failure
+
+on:
+  workflow_run:
+    workflows: ["Process new code merged to main"]
```

Comment:
Can we add this workflow name to some kind of constant/variable? I am worried that if the workflow changes its name, this will stop working.

Reply:
Sure, no worries, but I have a question, do you know if this will fail if for some reason the original workflow changes its name? I mean, I want it to fail, I just don't want to silently move on without us noticing.